### DISCUSSION TITLE
[npm] Support pnpm overrides

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#26249] [Suggest an Idea]
### Tell us more.

`pnpm.overrides`[^1] should be supported.

**`package.json`**
```json
{
  "pnpm": {
    "overrides": {
      "foo": "^1.0.0",
      "quux": "npm:@myorg/quux@^1.0.0",
      "bar@^2.1.0": "3.0.0",
      "qar@1>zoo": "2"
    }
  }
}
```

[^1]: https://pnpm.io/package_json#pnpmoverrides

**Comment by viceice** [comment]
Workaround: use `resolutions`[^1]

[^1]: https://pnpm.io/package_json#resolutions

**Comment by rarkins** [comment]
Should we create an issue for this?

**Reply by viceice** [reply]
yes please

**Reply by rarkins** [reply]
https://github.com/renovatebot/renovate/issues/26320
